### ISSUE TITLE
Patch selection broken in parallel for a silo amr dataset

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1553]
Dataset from Brian Ryujin

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 09/20/2010 06:19 pm
Original update: 09/23/2010 03:50 pm
Ticket number: 397
